tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -219,6 +219,7 @@ def wait_for_done(self, return_contacts=True):
             from gaiatest.apps.contacts.app import Contacts
             return Contacts(self.marionette)
         else:
-            Wait(self.marionette).until(lambda m: self.apps.displayed_app.name != 'Communications')
+            from gaiatest.apps.contacts.app import Contacts
+            Contacts(self.marionette).wait_to_not_be_displayed()
```

Comment:
Does this work with http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/functional/dialer/test_dialer_add_contact.py ?
I suspect it doesn't, in that case this needs the entry_point work from https://bugzilla.mozilla.org/show_bug.cgi?id=1207098 .